transmitter.js

```diff
@@ -1,16 +1,9 @@
-let transmissionFailureCount = 0;
-
-function networkTransmitStub(celcius) {
-    console.log(`Temperature to transmit: ${celcius} celcius`);
+const {networkTransmitStub} = require("./stub")

```

Comment:
Separation of test and production code: You've separated the files well: `stub.js` and `transmitter_stub_test.js` have test code, while this file is production code.
However, this file requires `stub.js`, which means consumers who need this functionality will need the stub as well. Think of a way to avoid this dependency. Hint: Lookup "dependency injection"

Reply:
I have made changes. Now the `transmitter_stub_test.js` can set the function `networkTransmit` (in file transmitter.js) using the function `setNetworktransmit` 

Can you please review it.